plugins/syntax/g_numeral.mlg

```diff
@@ -37,6 +37,9 @@ END
 VERNAC COMMAND EXTEND NumeralNotation CLASSIFIED AS SIDEFF
   | #[ locality = Attributes.locality; ] [ "Numeral" "Notation" reference(ty) reference(f) reference(g) ":"
       ident(sc) numnotoption(o) ] ->
-    { let (sigma, env) = Pfedit.get_current_context () in
+    { (* Global env is used here on purpose, in fact the type
```

Comment:
This change is orthogonal to this PR, let's move it to a separate one. I can do it if you like. Same for string notations below.